consensus/istanbul/backend/backend_test.go

```diff
@@ -63,7 +56,8 @@ func TestBackend_GetTargetReceivers(t *testing.T) {
 	configItems = append(configItems, epoch(3))
 	configItems = append(configItems, governanceMode("single"))
 	configItems = append(configItems, minimumStake(new(big.Int).SetUint64(4000000)))
-	configItems = append(configItems, istanbulCompatibleBlock(new(big.Int).SetUint64(5)))
```

Comment:
Why is this being removed?

Reply:
Thanks, applied at https://github.com/kaiachain/kaia/pull/557/commits/4e8b7dd90585dbb363e9307691c53397c1b80ace